ecl/hql/hqllex.l

```diff
@@ -1333,7 +1333,8 @@ BITFIELD{digit}+    {
                          size = 1;
                       }
 
-                      ITypeInfo * type = makeBitfieldType(size);
+                      Owned<ITypeInfo> int64 = makeIntType(8, false);
+                      ITypeInfo * type = makeBitfieldType(size, int64);
```

Comment:
I changed it because otherwise a bitfield 7 followed by a bitfield 9 do not pack into the same space.  (Similarly 1 followed by 63).  It make much more sense to have a default width, and then allow the user to override it.